docs/projects/fps/tutorial.md

```diff
@@ -240,79 +385,90 @@ namespace Fps
 }
 ```
 
-This script is mostly standard C# code that you could find in any game built with Unity Engine. There are a few annotations which are specific to the SpatialOS GDK though, let's break those down:
+This script is mostly standard C# code that you could find in any game built with Unity Engine. There are a few parts which are specific to the SpatialOS GDK though, let's break those down:
 
-* `using Pickups;`<br>
-The class `HealthPickupClientVisibility` relies upon the `is_active` property that we [previously](#define-a-new-spatialos-component) defined as a property of the `HealthPickup` component. For this reason the package `Pickups`, that we declared in the `health_pickup.schema` file, appears in a `using` statement at the top of the script.
+* `[WorkerType(WorkerUtils.UnityClient)]`<br/>
+This `WorkerType` annotation marks this `MonoBehaviours` to only be enabled for a specific worker-type. In this case, this `Monobehaviour` will only be enabled on `UnityClient` client-workers, ensuring that it will never run on your server-workers.
```

Comment:
```suggestion
This `WorkerType` annotation marks this `MonoBehaviours` to only be enabled for a specific worker-type. In this case, this `MonoBehaviour` will only be enabled on `UnityClient` client-workers, ensuring that it will never run on your server-workers.
```